### PR TITLE
Adding pyfilemanager v1.0.0 from pypi

### PR DESCRIPTION
**Pull request by praneethnamburi**
pyfilemanager is a simple tool for file search and file path management. It is currently available on PyPI. For more details, follow the links below.

[src](https://github.com/praneethnamburi/pyfilemanager)
[PyPI - Version](https://pypi.org/project/pyfilemanager/)
[Documentation Status](https://pyfilemanager.readthedocs.io)
[GitHub license](https://raw.githubusercontent.com/praneethnamburi/pyfilemanager/main/LICENSE)

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by praneethnamburi** [comment]
HI @conda-forge/help-python. This pull request is ready for review.

**Comment by praneethnamburi** [comment]
@conda-forge-admin, can you please ping @conda-forge/staged-recipes or @conda-forge/help-python saying that this pull request is ready for review.

File: recipes/pyfilemanager/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  sha256: da224c2f4658d0005a842804e116c7eada10b97bafd4b60d634467bf13b05424
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python <4.0
```
Comment (xhochy) [inline recipes/pyfilemanager/meta.yaml:19]: This should match the pin in run
```suggestion
    - python >=3.7,<4.0
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/pyfilemanager/meta.yaml (hunk 1 above)
Comment (praneethnamburi) [reply recipes/pyfilemanager/meta.yaml]: Thank you for reviewing my recipe, @xhochy. I made this change, and updated the package to v1.1.0 (latest version on pypi). The hash is also updated.

Comment (praneethnamburi) [reply recipes/pyfilemanager/meta.yaml]: I updated staged-recipes on my fork, and I see other recipes in there. Sorry if I shouldn't have done that. Perhaps this is the reason some of the checks are currently failing?

**Review by xhochy — APPROVED** [review]

**Pull request merged**